.github/workflows/wingsdk-mutation.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         if: steps.token-check.outputs.HAS_TOKEN == 'true'
         name: Find mutations
         run: |-
-          git add .
+          git add -u
```

Comment:
This doesn't cover new files, not sure it should.

Reply:
Maybe `-all` is better

Reply:
Introducing new snapshot files would be a valid use case for `--all`, let's try it